lntest/itest/lnd_test.go

```diff
@@ -10213,14 +10209,18 @@ func testMultiHopHtlcLocalTimeout(net *lntest.NetworkHarness, t *harnessTest) {
 		t.Fatalf("bob should have pending htlc but doesn't")
 	}
 
-	// Now we'll mine an additional block, which should include the second
-	// layer sweep tx.
-	block := mineBlocks(t, net, 1, 1)[0]
+	// Now we'll mine an additional block, which should confirm Bob's commit
+	// sweep. This block should also prompt Bob to broadcast their second
+	// layer sweep.
```

Comment:
I feel that the test would have been slightly better if we didn't use a zero outgoing broadcast delta. But I think out of scope for this pr.

Reply:
Actually, `DefaultOutgoingBroadcastDelta` is also zero, so we are testing what happens in production, which is good.